samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+namespace BinaryClasification_TitanicSurvivalPrediction
+{
+
+    internal static class Program
+    {
+        private static string AppPath => Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+        private static string TrainDataPath => Path.Combine(AppPath, "datasets", "titanic-train.csv");
+        private static string TestDataPath => Path.Combine(AppPath, "datasets", "titanic-test.csv");
+        private static string ModelPath => Path.Combine(AppPath, "TitanicModel.zip");
+
+        private static async Task Main(string[] args)
+        {
+            // STEP 1: Create a model
+            var model = await TrainAsync();
+
+            // STEP2: Test accuracy
+            Evaluate(model);
+
+            // STEP 3: Make a prediction
+            var prediction = model.Predict(TestTitanicData.Passenger);
+            Console.WriteLine($"Did this passenger survive? {(prediction.Survived ? "Yes" : "No")}");
```

Comment:
Would also suggest to word it "Did this passenger survive? Predicted: ... Actual : ..."
And just to be on the optimistic note maybe we can pick a one that survived? :) But that's only my preference.

Reply:
Fixed in bebf34c

Reply:
Thanks for the comments! It looks better now....